index.js

```diff
@@ -1,8 +1,67 @@
-// Iteration 1: Names and Input
 console.log("I'm ready!");
 
+// Iteration 1: Names and Input
+
+let Hacker1 = "Erik";
+//console.log(`The driver name is ${Hacker1}!`)
+
+let Hacker2 = "Francisco";
+//console.log(`The navigator name is ${Hacker2}!`)
+
 // Iteration 2: Conditionals
 
+if (Hacker1.length > Hacker2.length) {
+
+    console.log(`The driver has the longest name, it has ${Hacker1.length} characters.`);
+}
+else if (Hacker2.length > Hacker1.length){
+    console.log(`It seems that the navigator has the longest name, it has ${Hacker2.length} characters.`);
+}
+else {
+    console.log(`Wow, you both have equally long names, ${Hacker2.length} characters!`)
+}
+
 
 // Iteration 3: Loops
 
+let driver = ""
+for (let i = 0; i < Hacker1.length; i++){ 
+    driver = driver + Hacker1[i] + " ";
+
+}
+console.log(driver.toUpperCase());
+
+
+let navigator = ""
+for (let f = Hacker2.length-1; f >= 0; f--){
+    navigator = navigator + Hacker2[f];
+}
+
+console.log(navigator);
+
+
+
+if (Hacker2.localeCompare(Hacker1) === 1) {
+    console.log("The driver's name goes first.")
+}else if (Hacker1.localeCompare(Hacker2) === 1) {
```

Comment:
`.localCompare()` returns 1, 0 or -1 so here your condition should be checking if it is equal to -1:

```
( ... )
else if (Hacker1.localeCompare(Hacker2) === -1) 
( ... )
```